furnace/Author.py

```
@@ -53,20 +51,30 @@ def __repr__(self):
-                self._entity = False
+            with shelve.open(self.CACHE_FILE) as cache:
+                if self.s2_id:
+                    if self.s2_id in cache:
+                        r = cache[self.s2_id]
+                    else:
+                        if s2api is not None:
+                            headers = {
+                                'x-api-key': s2api
+                            }
+                        else:
+                            headers = None
```

Ruff (0.8.2)

Use ternary operator `headers = {'x-api-key': s2api} if s2api is not None else None` instead of `if`-`else`-block

Replace `if`-`else`-block with `headers = {'x-api-key': s2api} if s2api is not None else None`

(SIM108)